drivers/adodb-pdo_sqlsrv.inc.php

```diff
@@ -62,17 +62,12 @@ class ADORecordSet_pdo_sqlsrv extends ADORecordSet_pdo
 	public function fetchField($fieldOffset = 0)
 	{
 
-		static $fieldObjects = array();
+		$fieldObjects = array();
 		// Default behavior allows passing in of -1 offset, which crashes the method
 		if ($fieldOffset == -1) {
 			$fieldOffset++;
 		}
 
-		if (isset($fieldObjects[$fieldOffset])) {
-			// Look for cached field offset
```

Comment:
There are only two hard things in Computer Science: cache invalidation, naming things, and off by one errors.

Reply:
@mbrooks joke aside, are you actually trying to make a point here ?

Reply:
No, it was just a joke. I hardly ever get to use that line. But, in all seriousness, how can we make this work with the caching model currently in place in ADODB?

Reply:
@dregad It was just a joke. I hardly ever get to use that line. In all honesty, how can we help you with the cache problems?